source/includes/api/tables/progress-response.rst

```diff
@@ -51,13 +51,17 @@
    * - ``collectionCopy``
        ``.estimatedTotalBytes``
      - integer
-     - Estimated total number of bytes to be copied.
+     - Estimated total number of bytes globally to be copied.
```

Comment:
Consider either:

Estimated total number of bytes globally of all source clusters to be copied.
Estimated total number of bytes of all source clusters to be copied.

Reply:
Fixed